plugins/dra-ocp-validator/tests/test-dra-admin-access.sh

```
@@ -0,0 +1,165 @@
+#==============================================
+# Phase 1: Unauthorized Namespace
+#==============================================
+echo "=== PHASE 1: Unauthorized Namespace ==="
+NAMESPACE_UNAUTH="dra-no-admin"
+
+oc create namespace ${NAMESPACE_UNAUTH}
+echo "Created namespace without admin-access label"
+echo ""
+
+echo "--- Attempting ResourceClaim with adminAccess=true ---"
+cat <<EOF | tee ${LOG_DIR}/test1-claim.yaml | oc apply -f - 2>&1 | tee ${LOG_DIR}/test1-result.txt || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)